datamodel/src/main/java/com/hartwig/serve/datamodel/ActionableEvents.java

```diff
@@ -21,7 +22,10 @@ public abstract class ActionableEvents {
     public abstract List<ActionableHotspot> hotspots();
 
     @NotNull
-    public abstract List<ActionableRange> ranges();
+    public abstract List<ActionableCodon> codons();
+
+    @NotNull
+    public abstract List<ActionableExon> exons();
```

Comment:
What's the difference between an `ActionableCodon` and and `ActionableExon` in the way they're treated in the code, other than the fact that they belong to two different lists? I suggested implementing a common interface in another comment, but is there any reason not to use the same class? i.e.
```suggestion
    public abstract List<ActionableRange> codons();

    @NotNull
    public abstract List<ActionableRange> exons();
```

If codons and exons shared the same implementation but were stored in separate files and lists, I think we would get the same benefits without having to duplicate factories, converters, and comparators (and the corresponding tests).

Reply:
correct it can be use the same implementation and stored it in 2 separate files/lists